src/ImageSharp/Formats/Jpeg/Components/Block8x8F.cs

```diff
@@ -830,5 +830,46 @@ public void TransposeInto(ref Block8x8F d)
                 d.V7R.W = this.V7R.W;
             }
         }
+
+        /// <summary>
+        /// Compares entire 8x8 block to a single scalar value.
+        /// </summary>
+        /// <param name="value">Value to compare to.</param>
+        public bool EqualsToScalar(int value)
```

Comment:
I don't like this method but it's a must. Current pipeline converts encoded tables directly to `Block8x8F` and we should check if table contains all 1's...